### PR TITLE
ld subscribeChannelGraph type

### PR DESCRIPTION
**Pull request by leon-do**

### CODE REVIEW

File: types/lnrpc.d.ts [hunk 1]
```diff
@@ -489,6 +489,35 @@ export interface ListPaymentsResponse {
   payments: Payment[];
 }
 
+export interface NodeUpdate {
+  addresses: string[];
+  identityKey: string;
+  globalFeatures: string;
```
Comment (cavanmflynn) [inline types/lnrpc.d.ts]: Going off the docs, this should be a `Buffer`. Is it being returned as a string?

File: types/lnrpc.d.ts [hunk 2]
```diff
@@ -489,6 +489,35 @@ export interface ListPaymentsResponse {
...
+  capacity: string;
+  routingPolicy: RoutingPolicy;
+  advertisingNode: string;
+  connectingNode: string;
+}
+
+export interface ClosedChannelUpdate {
+  chanId: string;
+  capacity: string;
+  closedHeight: string;
```
Comment (cavanmflynn) [inline types/lnrpc.d.ts]: Looks like this should be a number (going off the docs again).

**Comment by danielemiliogarcia** [comment]
let me know when this is finished, i will update ln-stats i was using :any for now

File: types/lnrpc.d.ts (hunk 1 above)
Comment (leon-do) [reply types/lnrpc.d.ts]: yeah, it's returning a string

File: types/lnrpc.d.ts (hunk 2 above)
Comment (leon-do) [reply types/lnrpc.d.ts]: also returning a string

**Pull request merged**